src/PullRequestsAPI.ts

```diff
@@ -57,15 +58,17 @@ class GithubPullRequestAPI implements PullRequestAPI {
       repo: this.repo,
       issue_number: issue_number,
       body: body
-    })
+    }).catch(error => console.log(`An error occurred creating a comment on pull request #${issue_number}`, error))
+      .then(() => console.log("Continuing"))
   }
 
   async deletePullRequestBranch(pullRequest: PullRequest) {
     await this.git.rest.git.deleteRef({
       owner: this.owner,
       repo: this.repo,
       ref: `heads/${pullRequest.head.ref}`
-    })
+    }).catch(error => console.log(`An error occurred deleting branch associated with request #${pullRequest.number} at heads/${pullRequest.head.ref}`, error))
+      .then(() => console.log("Continuing"))
```

Comment:
To log out a warning we can actually do `core.warning("delete request failed.")`

Reply:
Ahh good to know, i've updated to use core.